modules/snapshot/lambda.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_lambda_permission" "snapshot" {
+  statement_id = "AllowExecutionFromCloudWatchForSnapshots"
+  action = "lambda:InvokeFunction"
+  function_name = "${ aws_lambda_function.snapshot.arn }"
+  principal = "events.amazonaws.com"
+  source_arn = "${ aws_cloudwatch_event_rule.snapshot.arn }"
+}
+
+resource "aws_lambda_function" "snapshot" {
+  filename = "${ path.module }/../../tmp/snapshot.zip"
```

Comment:
error happen with * aws_lambda_function.snapshot: Unable to load "../../tmp/snapshot.zip": open  /../../tmp/snapshot.zip: no such file or directory